<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CameraPreview</class>
 <widget class="QMainWindow" name="CameraPreview">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>804</width>
    <height>539</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>9</y>
      <width>781</width>
      <height>471</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="webcamPreview">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="filterButtonOne">
         <property name="text">
          <string>Filter 1</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="filterButtonTwo">
         <property name="text">
          <string>Filter 2</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="filterButtonThree">
         <property name="text">
          <string>Filter 3</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="filterButtonFour">
         <property name="text">
          <string>Filter 4</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="filterButtonFive">
         <property name="text">
          <string>Filter 5</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>804</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
